agent/src/services/twitter/services.ts

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import * as dotenv from "dotenv";
+import { PdfService } from "@ai16z/plugin-node";
+import * as fs from "fs";
+
+// Load environment variables from .env file
+dotenv.config();
+
+// Define constants
+const BEARER_TOKEN = process.env.BEARER_TOKEN;
+const BASE_URL = "https://api.twitter.com/2/tweets/search/recent";
+
+/**
+ * Fetch recent tweets related to sports.
+ * @param maxResults Number of tweets to fetch (default: 10)
+ */
+
+// this function will get data from documnets for buliding custom knowledge base
+export const fetchDocuments = async (): Promise<string[]> => {
+    const pdfService = new PdfService();
+    const pdfFilePath = "C:/Users/Dell/eliza-1/agent/src/services/twitter/game.pdf";  //a dummy pdf
```

Comment:
no hard coded paths, node has __dirname if you want to give a relative path to this file
https://nodejs.org/docs/latest/api/globals.html#__dirname